Dashboard/Dashboard/Presentation/PrimaryCourseDashboardView.swift

```diff
@@ -0,0 +1,354 @@
+//
+//  PrimaryCourseDashboardView.swift
+//  Dashboard
+//
+//  Created by  Stepanok Ivan on 16.04.2024.
+//
+
+import SwiftUI
+import Core
+import Theme
+import Swinject
+
+public struct PrimaryCourseDashboardView<ProgramView: View>: View {
+    
+    @StateObject
+    private var viewModel: PrimaryCourseDashboardViewModel
+    private let router: DashboardRouter
+    private let config = Container.shared.resolve(ConfigProtocol.self)!
```

Comment:
Maybe we could add config to the PrimaryCourseDashboardViewModel in ScreenAssembly to be consistent with other models/views?